javascript-modal/styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+#trigger {
+  width: 12vw;
+  height: 5vh;
+  border: none;
+  border-radius: 7px;
+  font-size: 1.5rem;
+  color: #fff;
+  background-color: #4a4ae9;
+  box-shadow: 0 2px 4px black;
+}
+
+#trigger:hover {
+  cursor: pointer;
+}
+
+#trigger:focus {
+  display: none;
+}
+
+.modal-content {
+  text-align: center;
+  font-weight: bold;
+  font-size: 1.1rem;
+  position: fixed;
+  width: 25vw;
+  height: 25vh;
+  padding-top: 1rem;
+  top: 25%;
+  left: 35%;
+  display: none;
+  background-color: #fff;
+  z-index: 100;
+}
+
+.modal-content.off {
+  display: block;
+}
+
+.no {
+  width: 8vw;
+  height: 5vh;
+  border-radius: 7px;
+  font-size: 2rem;
+  text-transform: uppercase;
+  border: none;
+  color: #fff;
+  background-color: #eb5d5d;
+  box-shadow: 0 2px 4px black;
+  margin-top: 2rem;
+}
+
+.no:hover {
+  cursor: pointer;
+}
+
+.no:focus {
+  display: none;
+}
+
+.active {
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  pointer-events: none;
+}
+
+.active.off {
+  opacity: 1;
+}
```

Comment:
You should not need `opacity` for this. Instead, the active modal will be a `fixed` position `rgba()` element with a white box centered inside. Try `display: flex` on the black part and then center the white box inside.